Failed to install 'cordova-plugin-googleplus': Error: pod: Command failed with exit code 1

### Trying to add cordova platform IOS in ionic fails because of this plugin
```
MacBook-Pro$ ionic cordova platform add ios
> cordova platform add ios
...
Installing "cordova-plugin-googleplus" for ios
Running command: pod install --verbose
Failed to install 'cordova-plugin-googleplus': Error: pod: Command failed with exit code 1
    at ChildProcess.whenDone (/Users/.../node_modules/cordova-common/src/superspawn.js:135:23)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:962:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
Failed to restore plugin "cordova-plugin-googleplus" from config.xml. You might need to try adding it again. Error: Error: pod: Command failed with exit code 1
...
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "GoogleUtilities/UserDefaults":
  In snapshot (Podfile.lock):
    GoogleUtilities/UserDefaults (= 6.2.5, ~> 6.0, ~> 6.2)

  In Podfile:
    GoogleUtilities (= 6.2.3) was resolved to 6.2.3, which depends on
      GoogleUtilities/UserDefaults (= 6.2.3)

Specs satisfying the `GoogleUtilities/UserDefaults (= 6.2.5, ~> 6.0, ~> 6.2), GoogleUtilities/UserDefaults (= 6.2.3)` dependency were found, but they required a higher minimum deployment target.
```

### Environment details:
```
MacBook-Pro$ node -v
v10.12.0
MacBook-Pro$ ionic -v
5.2.3
MacBook-Pro$ cordova -v
9.0.0 (cordova-lib@9.0.1)
```

Please ask me if you need any more details.
Thanks in advance

> I just provided a PR to solve this #635
> 
> Till its merged you could install the plugin from my PR's branch:
> 
> ```
> ionic cordova plugin add https://github.com/peterpeterparker/cordova-plugin-googleplus#optimistic-cocoapods
> ```

Have you removed the plugin from the repo. I am getting error while adding it 

Probably this is either a connection problem, or plugin spec is incorrect.
Check your connection and plugin name/version/URL.
Error: npm: Command failed with exit code 1 Error output:
npm ERR! Error while executing:
npm ERR! /usr/bin/git ls-remote -h -t https://github.com/peterpeterparker/cordova-plugin-googleplus.git
npm ERR! 
npm ERR! remote: Repository not found.
npm ERR! fatal: repository 'https://github.com/peterpeterparker/cordova-plugin-googleplus.git/' not found
npm ERR! 
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jassi/.npm/_logs/2019-09-18T11_40_31_159Z-debug.log
[ERROR] An error occurred while running subprocess cordova.
        
        cordova plugin add https://github.com/peterpeterparker/cordova-plugin-googleplus... exited with exit code 1.

--- a/plugin.xml
+++ b/plugin.xml
@@ -97,7 +97,7 @@
       </config>
       <pods use-frameworks="true">
         <pod name="GoogleSignIn" spec="~> 4.4.0"/>
-        <pod name="GoogleUtilities" spec="~> 6.2.3"/>
+        <pod name="GoogleUtilities" spec="~> 6.3.0"/>
       </pods>
     </podspec>
 

